CONTRIBUTING.md

```diff
@@ -31,12 +66,7 @@ How to teach Moto to support a new AWS endpoint:
 * Implementing the feature itself can be done by creating a method called `import_certificate` in `moto/acm/responses.py`. It's considered good practice to deal with input/output formatting and validation in `responses.py`, and create a method `import_certificate` in `moto/acm/models.py` that handles the actual import logic.
 * If you can also implement the code that gets that test passing then great! If not, just ask the community for a hand and somebody will assist you.
 
-## Before pushing changes to GitHub
-
-1. Run `black moto/ tests/` over your code to ensure that it is properly formatted
-1. Run `make test` to ensure your tests are passing
-
-## Missing services
+# Missing services
```

Comment:
The problem with referencing the PRs in this section is that those PRs import some Python 2 libraries, `urls_base` doesn't have the escape characters in the urls regex and the old `backend.py` is referenced.  

I looked at one of these PRs when I was trying to add a new service and thought it looked outdated, so then I tried the scaffold script.  I dunno.  

It took me a while to figure out how to interact with `click` of the scaffold script and I almost gave up in frustration.  I think the tab key expands the current selection and the arrow keys move up/down the lists, but I'm not sure I'm remembering correctly.

Reply:
Agreed - I've updated one of the PR's to a more recent one. 
There haven't been any new services since the updated urls/backend, unfortunately, but when one comes along, I'll update this doc again.